gulpfile.js

```diff
@@ -116,14 +116,29 @@ gulp.task( 'translations:download', () => {
 
 // Releasing. -----------------------------------------------------------------
 
-gulp.task( 'changelog:dependencies', () => {
+gulp.task( 'changelog', () => {
 	assertIsInstalled( '@ckeditor/ckeditor5-dev-env' );
 
-	return require( '@ckeditor/ckeditor5-dev-env' )
-		.generateChangelogForSubRepositories( {
-			cwd: process.cwd(),
-			packages: 'packages'
-		} );
+	const devEnv = require( '@ckeditor/ckeditor5-dev-env' );
+	const commonOptions = {
+		cwd: process.cwd(),
+		packages: 'packages'
+	};
+
+	const optionsForDependencies = Object.assign( {}, commonOptions, {
+		skipPackages: [
+			'@ckeditor/ckeditor5-build-inline',
+			'@ckeditor/ckeditor5-build-classic',
+			'@ckeditor/ckeditor5-build-balloon'
+		]
+	} );
+	const optionsForBuilds = Object.assign( {}, commonOptions, {
+		scope: /-build-/
```

Comment:
Always make the regexp as specific as possible, but not more. This one is going to fail if we ever have a package with `-build-` used anywhere (which is rather improbable but why risking anyway).